controllers/slicegateway/slicegateway.go

```diff
@@ -534,7 +564,7 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
 	return dep
 }
 
-func (r *SliceGwReconciler) GetGwPodNameAndIP(ctx context.Context, sliceGw *kubeslicev1beta1.SliceGateway) (string, string) {
+func (r *SliceGwReconciler) GetGwPodNameAndIP(ctx context.Context, sliceGw *kubeslicev1beta1.SliceGateway) ([]string, []string) {
```

Comment:
Would prefer to return an array of struct. Define a local struct that holds the pod name and IP. The struct could be extended with more attributes if needed in the future.